src/transformation/Transformation.ts

```diff
@@ -41,6 +41,11 @@ class Transformation {
     return this;
   }
 
+  addRawAction(raw: string) {
```

Comment:
this is unrelated to concatenate right?

Reply:
Somewhat related,
I had to include it to support this syntax:
`concatSourceTx.addRawAction(this.getTransitionString());`

We add a custom action to the source passed to concatenate.